.gitlab/pipeline/zombienet/polkadot.yml

```diff
@@ -161,6 +161,8 @@ zombienet-polkadot-functional-0011-async-backing-6-seconds-rate:
 zombienet-polkadot-elastic-scaling-0001-basic-3cores-6s-blocks:
   extends:
     - .zombienet-polkadot-common
+  variables:
+    FORCED_INFRA_INSTANCE: "spot-iops"
```

Comment:
We should use better naming, something like `high-iops-instance` does more justice

Reply:
cc @pepoviola